public/src/interfaces/User.ts

```diff
@@ -1,6 +1,12 @@
+export enum AuthType {
+  FORM_BASED = 'application/x-www-form-urlencoded',
+  APPLICATION_JSON = 'application/json'
```

Comment:
Use a more abstract way to reflect auth type, especially when you have `FormMode`
e.g.

```ts
export enum AuthType {
  FORM = 'form',
  // OIDC, NTLM, etc
}
```